bin/check-container-logs.rb

```diff
@@ -37,36 +48,34 @@
 
 class ContainerLogChecker < Sensu::Plugin::Check::CLI
   option :docker_host,
-         description: 'Docker socket to connect. TCP: "host:port" or Unix: "/path/to/docker.sock" (default: "127.0.0.1:2375")',
+         description: 'Docker API URI. https://host, https://host:port, http://host, http://host:port, host:port, unix:///path
+         Default: ENV[DOCKER_URL] => ENV[DOCKER_HOST] => /var/run/docker.sock',
```

Comment:
I believe what you want here is `ENV[DOCKER_URL]`

Reply:
It is not the default value used by the script itself but just here to explain how DockerApi class choose the [default host](https://github.com/rockandska/sensu-plugins-docker/blob/1590252ca5f25826c4684079df5b65e7b770bf64/lib/sensu-plugins-docker/client_helpers.rb#L7) if none is provided.
Maybe I should had more spaces to align the "Default" word with the word "Docker" on the previous line to let it be readable in the code without think it is the "default" for mixlib ?  

It is display in this form, maybe not elegant at it should be but not found a convenient way to indent a multiple line description.

````
$ bundle exec bin/metrics-docker-stats.rb --help
Usage: bin/metrics-docker-stats.rb (options)
    -N, --container-name CONTAINER   Name of container to collect metrics for
    -P, --percentage                 add cpu usage percentage metric
    -d, --deliminator                the deliminator to use with -m
    -H, --docker-host DOCKER_HOST    Docker API URI. https://host, https://host:port, http://host, http://host:port, host:port, unix:///path
         Default: ENV[DOCKER_URL] => ENV[DOCKER_HOST] => /var/run/docker.sock
    -e, --environment-tags ENV_VARS  Name of environment variables on each container to be appended to metric name, separated by commas
    -n, --names                      use friendly name if available
    -i, --ioinfo                     enable IO Docker metrics
    -m, --match index                Partial names by spliting and returning at index(es).
         eg. -m 3,4 my-docker-container-process_name-b2ffdab8f1aceae85300 for process_name.b2ffdab8f1aceae85300
    -s, --scheme SCHEME              Metric naming scheme, text to prepend to metric
````

If you have a better idea for the indentation or a place where to explain how the default host is choose i'm open to hear it.

Reply:
I think the best place to explain the defaults are in the readme, similar to [this](https://github.com/sensu-plugins/sensu-plugins-aws/tree/10.2.0#authentication)